06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md

```diff
@@ -0,0 +1,6 @@
+#### Output
+- **Using**: TravisCI
+- **Deliver**: a link to a successful TravisCI build of your `development` branch (it should include test runs)
+- **With**: *independently*
+
+#### [Link to Successful Travis Build](https://travis-ci.org/andela-aawa/inverted-index)
```

Comment:
Even though this build passes, your latest build has failed. Please be mindful of this moving forward.